src/teleop/gui/src/components/JointStateTable.vue

```diff
@@ -0,0 +1,180 @@
+<template>
+<div class="wrap">
+    <div v-if="vertical">
+        <div>
+            <h3> Motor Data </h3>
+        </div>
+        <table class="tableFormat" style="undefined;table-layout: fixed; width: 745px">
+            <colgroup>
+                <col style="width: 85px">
+                <col style="width: 60px">
+                <col style="width: 75px">
+                <col style="width: 85px">
+                <col style="width: 60px">
+                <col style="width: 75px">
+                <col style="width: 85px">
+                <col style="width: 60px">
+                <col style="width: 75px">
+                <col style="width: 85px">
+            </colgroup>
+
+            <thead>
+
+                <tr class="Bold">
+                    <th class="tableElement">Motor</th>
+                    <th class="tableElement">Positon (m)</th>
+                    <th class="tableElement">Velocity (m/s)</th>
+                    <th class="tableElement">Effort (Nm)</th>
+
+                </tr>
+            </thead>
+
+            <tbody>
+
+                <tr v-for="motor in motors">
+                    <th class="tableElement">{{motor.name}}</th>
+                    <td class="tableElement">{{motor.position}} </td>
+                    <td class="tableElement">{{motor.velocity}} </td>
+                    <td class="tableElement">{{motor.effort}} </td>
+
+                </tr>
+            </tbody>
+        </table>
+    </div>
+
+    <div v-else>
+        <div>
+            <h3> Motor Data </h3>
+        </div>
+        <table class="tableFormat" style="undefined;table-layout: fixed; width: 745px">
+            <colgroup>
+                <col style="width: 85px">
+                <col style="width: 60px">
+                <col style="width: 75px">
+                <col style="width: 85px">
+                <col style="width: 60px">
+                <col style="width: 75px">
+                <col style="width: 85px">
+                <col style="width: 60px">
+                <col style="width: 75px">
+                <col style="width: 85px">
+            </colgroup>
+
+            <thead>
+
+                <tr class="Bold">
```

Comment:
Are you sure this style is being used? Maybe it is but at the very least we should make cases match to avoid confusing people in the future.